python_benchmarks/qr_codes/qr.py

```diff
@@ -188,17 +213,24 @@ def main():
     gl_count = 0
     gl_detect = 0
     gl_decode = 0
+    gl_dist = 0
+    gl_pos_dist = 0
     qr = create_instance_qr(DetectorQR.TypeDetector[algorithm], model_path)
     for dir in list_dirs:
         imgs_path = find_images_path(dir)
         qr_count = 0
         qr_detect = 0
         qr_decode = 0
+        category_dist = 0
+        category_pos_dist = 0
         for img_path in imgs_path:
             label_path = img_path[:-3] + "txt"
             gold_corners = get_gold_corners(label_path)
             qr_count += gold_corners.shape[0]
             image = cv.imread(img_path, cv.IMREAD_IGNORE_ORIENTATION)
+            empty_img = np.zeros(image.shape, dtype=np.uint8)
+            empty_img2 = np.zeros(image.shape, dtype=np.uint8)
```

Comment:
Are the images detection masks? Please name them accordingly. Do we need them outside of `get_norm_to_rotate_qr`?

Reply:
rename `empty_img` to `mask`
remove outside  variable declaration